### ISSUE TITLE
USB Log files should not be split every time the robot gets disabled

### ISSUE THREAD

**Issue by ewpratten** [frc5024/lib5k#1]
Currently, due to https://github.com/frc5024/lib5k/blob/master/logging/src/main/java/io/github/frc5024/lib5k/logging/USBLogger.java#L97-L104, the USB log files are split every time the robot gets disabled. 

While in theory, this is a good idea, brownouts and unreliable event networks can cause logs to split at the wrong time. Instead, we should not split at all, and let the file naturally split when the robot is shut off.

Basically, we just need to remove those lines, and test to make sure its all good

**Comment by ewpratten** [comment]
@catarinaburghi are you able to fix and test this?

**Comment by catarinaburghi** [comment]
I am unfortunately unable to do this due to summer school. I would love to help but I am extremely busy. Sorry.

**Comment by ewpratten** [comment]
fixed
